src/app.js

```diff
@@ -21,13 +22,16 @@ export function app(app) {
     })
   }
 
-  if (document.readyState[0] !== "l") {
-    load()
-  } else {
-    addEventListener("DOMContentLoaded", load)
-  }
+  return emit("loaded", emit, render(state, view))
```

Comment:
Haha, that is fine.. you are right. I don't do it often. Some people might, for them it will break so I guess some explicit *makes sure to put the script before the closing body tag* wording in the docs wouldn't go a miss.